### ISSUE TITLE
Privacy-Settings

### ISSUE THREAD

**Issue by mpipoly** [brainhack-uiowa/uibrainhack#20]
Hey @jdkent, I am not sure that I am able to change privacy settings on some accounts that we share. Is this something you can easily change? I am specifically talking about this folder: https://github.com/brainhack-uiowa/uibrainhack

But... if there are other folders where myself and/or other Brainhack-Hackyhour folks don't have access/control, could you also change that as well?

**Comment by jdkent** [comment]
I changed you to an owner within this organization, so you can change the visibility (https://github.com/brainhack-uiowa/uibrainhack/settings#danger-zone), do the files you added contain sensitive information?

**Comment by mpipoly** [comment]
@jdkent Yes, some of the files have first and last names (2019 Attendance) as well as other identifiable information from the last two Brainhack's. Additionally, I recall some other personal information that may have been provided in our written feedback from last year. I thought it best to prevent non-committee members from seeing our pre and post-Brainhack data and response information. I can transcribe the summary info though and make that public if you or other members think there is value in making the non-identifiable information public.
